components/at_mention/at_mention.tsx

```diff
@@ -45,9 +56,13 @@ export default class AtMention extends React.PureComponent {
         this.overlayRef = React.createRef();
     }
 
-    handleClick = (e) => {
-        const targetBounds = this.overlayRef.current.getBoundingClientRect();
-        const placement = popOverOverlayPosition(targetBounds, window.innerHeight, spaceRequiredForPopOver);
+    handleClick = (e: React.MouseEvent) => {
+        const targetBounds = this.overlayRef.current?.getBoundingClientRect();
+
+        let placement;
+        if (targetBounds) {
+            placement = popOverOverlayPosition(targetBounds, window.innerHeight, spaceRequiredForPopOver);
+        }
 
         this.setState({target: e.target, show: !this.state.show, placement});
```

Comment:
this needs to be adjusted, since we call `setState` regardless of a value set for `placement` or not. This could potentially lead to bugs.